vision/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
 
 [1]: https://pypi.org/project/google-cloud-vision/#history
 
+## 0.41.0
+
+12-05-2019 14:49 PST
+
+- fix(vision): deprecate resource name helper methods (via synth) ([#9866](https://github.com/googleapis/google-cloud-python/pull/9866))
```

Comment:
Would you mind deleting this entry and the empty 'Implementation Changes' section?

Thank you!

Reply:
Sure! my b :stuck_out_tongue_closed_eyes: